source/generators/MuonAngleGenerator.h

```diff
@@ -71,12 +78,22 @@ namespace nexus {
     G4String ang_file_; ///< Name of file with distributions
     G4String dist_name_; ///< Name of distribution in file
 
-    TH2F * distribution_; ///< Anglular distribution
-
     const GeometryBase* geom_; ///< Pointer to the detector geometry
 
     G4VSolid * geom_solid_;
 
+    std::vector<G4double> flux, azimuths, zeniths; ///< Values of flux, azimuth and zenith from file
+    std::vector<G4double> azimuth_bins; ///< List of Azimuth bin edges
+    std::vector<G4double> zenith_bins;  ///< List of Zenith bin edges
+    std::vector<G4double> az_BW; ///< List of Azimuth bin widths
+    std::vector<G4double> zen_BW; ///< List of Zenith bin widths
+
+    std::mt19937 RN_engine;  ///< Random Number Generator Engine Index
+    std::mt19937 RN_engine_az; ///< Random Number Generator Engine Azimuth
+    std::mt19937 RN_engine_zen; ///< Random Number Generator Engine Zenith
+
+    std::discrete_distribution<G4int> discr_dist; ///< Discrete distribution to sample flux
```

Comment:
By convention, data members in nexus are followed by a trailing underscore, see the [Coding conventions and style guide](https://github.com/next-exp/nexus/wiki/Coding-conventions-and-style-guide)

Reply:
Thanks for bringing this to my attention, I was not aware of this practice and will update the variable names to match this style.